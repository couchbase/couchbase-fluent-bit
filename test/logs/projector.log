2021-03-09T17:32:15.064+00:00 [Info] Projector started with command line: [/opt/couchbase/bin/projector --httpsPort=9999 --certFile=/opt/couchbase/var/lib/couchbase/config/memcached-cert.pem --keyFile=/opt/couchbase/var/lib/couchbase/config/memcached-key.pem -kvaddrs=127.0.0.1:11210 -adminport=:9999 -diagDir=/opt/couchbase/var/lib/couchbase/crash -ipv6=false 127.0.0.1:8091]
2021-03-09T17:32:15.145+00:00 [Info] PROJ[:9999] started ...
2021-03-09T17:32:15.145+00:00 [Info] pram[:9999] registered /adminport/vbmapRequest
2021-03-09T17:32:15.146+00:00 [Info] MSAT starting with 60000 tick ...
2021-03-09T17:32:15.148+00:00 [Info] New settings received: 
{"indexer.settings.compaction.days_of_week":"Sunday,Monday,Tuesday,Wednesday,Thursday,Friday,Saturday","indexer.settings.compaction.interval":"00:00,00:00","indexer.settings.persisted_snapshot.interval":5000,"indexer.settings.log_level":"info","indexer.settings.compaction.compaction_mode":"circular","indexer.settings.compaction.min_frag":30,"indexer.settings.inmemory_snapshot.interval":200,"indexer.settings.max_cpu_percent":0,"indexer.settings.storage_mode":"memory_optimized","indexer.settings.recovery.max_rollbacks":2,"indexer.settings.memory_quota":268435456,"indexer.settings.compaction.abort_exceed_interval":false}
2021-03-09T17:32:15.149+00:00 [Warn] PROJ[:9999] immutable settings queryport.client.settings.poolSize cannot be update to `5000`
2021-03-09T17:32:15.151+00:00 [Info] amd64 linux; cpus: 6; GOMAXPROCS: 4; version: go1.11.6
2021-03-09T17:32:15.157+00:00 [Info] PROJ[:9999] ##1 doShutdownTopic() "MAINT_STREAM_TOPIC_6fc08c5818ffa52dc6c624eda31bddc1"
2021-03-09T17:32:15.158+00:00 [Error] PROJ[:9999] ##1 acquireFeed(): projector.topicMissing
2021-03-09T17:34:15.016+00:00 [Info] memstats {"Alloc":3799112, "TotalAlloc":10080160, "Sys":72284408, "Lookups":0, "Mallocs":84202,"Frees":66283, "HeapAlloc":3799112, "HeapSys":66224128, "HeapIdle":61046784, "HeapInuse":5177344,"HeapReleased":0, "HeapObjects":17919,"GCSys":2371584, "LastGC":1615311232357410700,"PauseTotalNs":31967700, "PauseNs":[370200, 16736500, 11795000], "NumGC":5}
