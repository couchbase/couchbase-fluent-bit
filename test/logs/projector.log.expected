/fluent-bit/test/logs/projector.log.actual: [1615311135.064000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.064+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"Projector started with command line: [/opt/couchbase/bin/projector --httpsPort=9999 --certFile=/opt/couchbase/var/lib/couchbase/config/memcached-cert.pem --keyFile=/opt/couchbase/var/lib/couchbase/config/memcached-key.pem -kvaddrs=127.0.0.1:11210 -adminport=:9999 -diagDir=/opt/couchbase/var/lib/couchbase/crash -ipv6=false 127.0.0.1:8091]","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.145000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.145+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"PROJ[:9999] started ...","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.145000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.145+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"pram[:9999] registered /adminport/vbmapRequest","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.146000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.146+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"MSAT starting with 60000 tick ...","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.148000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.148+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"New settings received: \n{\"indexer.settings.compaction.days_of_week\":\"Sunday,Monday,Tuesday,Wednesday,Thursday,Friday,Saturday\",\"indexer.settings.compaction.interval\":\"00:00,00:00\",\"indexer.settings.persisted_snapshot.interval\":5000,\"indexer.settings.log_level\":\"info\",\"indexer.settings.compaction.compaction_mode\":\"circular\",\"indexer.settings.compaction.min_frag\":30,\"indexer.settings.inmemory_snapshot.interval\":200,\"indexer.settings.max_cpu_percent\":0,\"indexer.settings.storage_mode\":\"memory_optimized\",\"indexer.settings.recovery.max_rollbacks\":2,\"indexer.settings.memory_quota\":268435456,\"indexer.settings.compaction.abort_exceed_interval\":false}","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.149000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.149+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"PROJ[:9999] immutable settings queryport.client.settings.poolSize cannot be update to `5000`","level":"WARN","level_numeric":"30"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.151000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.151+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"amd64 linux; cpus: 6; GOMAXPROCS: 4; version: go1.11.6","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.157000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.157+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"PROJ[:9999] ##1 doShutdownTopic() \"MAINT_STREAM_TOPIC_6fc08c5818ffa52dc6c624eda31bddc1\"","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/projector.log.actual: [1615311135.158000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:32:15.158+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"PROJ[:9999] ##1 acquireFeed(): projector.topicMissing","level":"ERROR","level_numeric":"40"}]
/fluent-bit/test/logs/projector.log.actual: [1615311255.016000000, {"file_path":"/fluent-bit/test/logs","file":"projector.log","fluentbit.version":"99","filename":"/fluent-bit/test/logs/projector.log","timestamp":"2021-03-09T17:34:15.016+00:00","couchbase":{"cluster":"test-cluster","node":"cb-node.test-cluster.default.svc","query":"false","search":"false","logging.version":"99","index":"false","operator.version":"unknown","server.version":"unknown","eventing":"false","node-config":"unknown","server.server":"unknown","analytics":"false","data":"false"},"hostname":"test-container","pod":{"uid":"unknown","namespace":"default","name":"test-pod"},"logshipper":"couchbase.sidecar.fluentbit","message":"memstats {\"Alloc\":3799112, \"TotalAlloc\":10080160, \"Sys\":72284408, \"Lookups\":0, \"Mallocs\":84202,\"Frees\":66283, \"HeapAlloc\":3799112, \"HeapSys\":66224128, \"HeapIdle\":61046784, \"HeapInuse\":5177344,\"HeapReleased\":0, \"HeapObjects\":17919,\"GCSys\":2371584, \"LastGC\":1615311232357410700,\"PauseTotalNs\":31967700, \"PauseNs\":[370200, 16736500, 11795000], \"NumGC\":5}","level":"INFO","level_numeric":"20"}]
